WalletWasabi/Tor/TorSettings.cs

```diff
@@ -76,13 +76,15 @@ public static string GetTorBinaryFilePath(OSPlatform? platform = null)
 
 	public string GetCmdArguments()
 	{
+		ControlEndpoint.TryGetPort(out var port);
+		
 		// `--SafeLogging 0` is useful for debugging to avoid "[scrubbed]" redactions in Tor log.
 		List<string> arguments = new()
 		{
 			$"--LogTimeGranularity 1",
 			$"--SOCKSPort \"{SocksEndpoint} ExtendedErrors\"",
 			$"--CookieAuthentication 1",
-			$"--ControlPort {ControlEndpoint.Port}",
+			$"--ControlPort {port ?? 9051}",
```

Comment:
Can you add explanation comment here?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/8851/files#r937085234 https://github.com/zkSNACKs/WalletWasabi/pull/8851